src/Chainweb/Cut/CutHashes.hs

```diff
@@ -93,3 +101,7 @@ cutToCutHashes p c = CutHashes
     , _cutHashesChainwebVersion = _chainwebVersion c
     }
 
+-- | The "Cut Height" represented by the given `CutHashes`.
+--
+cutHashesHeight :: CutHashes -> BlockHeight
+cutHashesHeight = view (cutHashes . each . _1)
```

Comment:
There is already a `_cutHashesHeight` field in `CutHashes. Also, do you need a `Lens` or would a `Getter` be enough?

Reply:
Woops I'm an idiot.